spec/promoted/ruby/client/request_builder_spec.rb

```diff
@@ -112,71 +112,77 @@
                     }
                   },
                   delivery_log: [{
+                    execution: {
+                      execution_server: "SDK",
+                      server_version: "rb." + Promoted::Ruby::Client::VERSION
+                    },
                     request: {
                       :user_info=> {:user_id=>"912", :log_user_id=>"91232"},
-                      :use_case=>"FEED",
                       device: {
                         device_type: "DESKTOP",
                         ip_address: "127.0.0.1",
                         browser: {
                             user_agent: "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36"
                         }
                       },  
+                      :use_case=>"FEED",
                       :properties=>
                       {
                         :struct=>
                         {
                           :query => {}
                         }
                       },
+                      client_request_id: "10",
                       request_id: "10",
-                      client_request_id: "10"
-                    },
-                    execution: {
-                      execution_server: "SDK",
-                      server_version: "rb." + Promoted::Ruby::Client::VERSION
+                      insertion:
```

Comment:
I'm not sure what I did to add request.insertions.  It usually makes more sense to have the request insertions.

Reply:
For some reason, we're now getting `DeliveryLog.request.insertion` filled in.  This is probably a good change.